setup.py

```diff
@@ -0,0 +1,41 @@
+# https://coderwall.com/p/qawuyq
+# Thanks James.
+
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except (IOError, ImportError):
+    long_description = ''
+    print 'warning: pandoc or pypandoc does not seem to be installed; using empty long_description'
+
+import os
+requirements_file = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'requirements.txt')
+with open(requirements_file, 'r') as f:
+    install_requires = [x.strip() for x in f.readlines()]
+
+import importlib
+from setuptools import setup
+
+setup(
+    name='metrics-py',
+    version=importlib.import_module('metrics').__version__,
+    author='Body Labs',
+    author_email='chenyang.liu@bodylabs.com',
+    description='Metrics send/query abstraction layer with URI-based backend configuration',
+    long_description=long_description,
+    url='https://github.com/bodylabs/metrics-py',
+    license='Apache',
```

Comment:
Can you grab our default license? See bodylabs/skeleton-py#1. Also need to update the LICENSE file.

(Unless there's some reason to prefer this one.)